examples/argocd-with-applications/apps/argocd.yaml

```diff
@@ -0,0 +1,111 @@
+"apiVersion": "argoproj.io/v1alpha1"
```

Comment:
Please remove the apps folder and sensitive data

Reply:
appsargoproj
common
hydrosphere